Firmware/mmu2_reporting.cpp

```diff
@@ -19,20 +23,47 @@ void EndReport(CommandInProgress cip, uint16_t ec) {
     custom_message_type = CustomMsg::Status;
 }
 
-void ReportErrorHook(CommandInProgress cip, uint16_t ec) {
+/**
+ * @brief Renders any characters that will be updated live on the MMU error screen.
+ *Currently, this is FINDA and Filament Sensor status and Extruder temperature.
+ */
+static void ReportErrorHookDynamicRender(void)
```

Comment:
side note - I like the separation of static and dynamic rendering, but for the final we may actually need to repaint the static text occasionally as well (in case of LCD glitches occur)